### PR TITLE
Update configure.sh

### PR DESCRIPTION
**Pull request by loumalouomega**
Correcting the configure.sh , incompatible with old processors

### CODE REVIEW

File: applications/MeshingApplication/custom_external_libraries/mmg/build/configure.sh [hunk 1]
```diff
@@ -4,8 +4,12 @@ cmake .. \
 -DSCOTCH_INCLUDE_DIR="/usr/include/scotch/"	            \
 -DSCOTCH_LIBRARY="libscotchmetis-5.1.so"	            \
 -DSCOTCHERR_LIBRARY="libscotcherr-5.1.so"	            \
--DCMAKE_CXX_FLAGS="-O3 -mavx2 -fPIC -fopenmp"               \
--DCMAKE_C_FLAGS="-O3 -mavx2 -fPIC -fopenmp"                 \
+-DCMAKE_CXX_FLAGS="-O3 -fPIC -fopenmp"               \
+-DCMAKE_C_FLAGS="-O3 -fPIC -fopenmp"                 \
```
Comment (roigcarlo) [inline applications/MeshingApplication/custom_external_libraries/mmg/build/configure.sh]: You should try to use at least a less modern set of SIMD instructions if AVX2 is not available in your processor.

Please see (top older, bottom newer):
https://software.intel.com/sites/landingpage/IntrinsicsGuide/

Comment (RiccardoRossi) [reply applications/MeshingApplication/custom_external_libraries/mmg/build/configure.sh]: just to make it clear, we should have something that is very compatible, so i would not push any flag which refers to a specific set of instructions

Comment (roigcarlo) [reply applications/MeshingApplication/custom_external_libraries/mmg/build/configure.sh]: Then we should remove it from the example configure as well.

Comment (roigcarlo) [reply applications/MeshingApplication/custom_external_libraries/mmg/build/configure.sh]: It seems that pretty much everything supports at least `-msse3`. Even non-intel/AMD processors in some cases.

https://en.wikipedia.org/wiki/SSE3

**Comment by RiccardoRossi** [comment]
One question:

what happens in windows?

**Comment by roigcarlo** [comment]
You have to enable them using `/arch:AVX2` for instance.  As of VS 2015 there is support up to AVX2, but not AVX-512

The question would be, does CMake takes care of making the conversion from `-mxxx` to `/arch:xxx` ?

**Comment by loumalouomega** [comment]
Effectively, the changes suggested by @roigcarlo  are working

**Comment by loumalouomega** [comment]
Changes done

**Review by RiccardoRossi — APPROVED** [review]

**Pull request merged**